src/solvers/dgsem_tree/container_viscous_2d.jl

```diff
@@ -0,0 +1,62 @@
+mutable struct ViscousContainer2D{uEltype <: Real}
+    u_transformed::Array{uEltype, 4}
+    # IDEA: Use SVector for fixed sized vectors?
```

Comment:
I could not get this to work, however - maybe because of the re-init?

Reply:
How did it fail specifically?

Reply:
I think ultimately the reason behind this is that it does not suffice for the outer dimensions to stay the same, but that the actual size of the entries is fixed - which is within the AMR scope not the case.